packages/styling/src/styles/UtilityStyles.ts

```diff
@@ -0,0 +1,12 @@
+import { IRawStyle } from '../interfaces/index';
+
+export const hideText: IRawStyle = {
+  position: 'absolute',
+  width: '1px',
+  height: '1px',
+  margin: '-1px',
```

Comment:
drat...this code shouldn't be in this PR. As for that line, i just moved it, didn't write it :)